addons/stock/views/res_config_settings_views.xml

```diff
@@ -113,7 +113,7 @@
                                     </div>
                                 </div>
                             </setting>
-                            <setting id="manage_product_packaging" title="Manage product packagings (e.g. pack of 6 bottles, box of 10 pieces)" help="Manage product packagings (e.g. pack of 6 bottles, box of 10 pieces)" documentation="/applications/inventory_and_mrp/inventory/management/products/usage.html#packaging">
+                            <setting id="manage_product_packaging" title="Packagings" help="Manage product packagings (e.g. pack of 6 bottles, box of 10 pieces)" documentation="/applications/inventory_and_mrp/inventory/product_management/product_tracking/packaging.html">
```

Comment:
Do you have to change the title in stable? I think it will break the translation for it for all existing dbs

Reply:
Thank you for pointing out that it'll break translations. In that case, let's not change the title-- just the link!